imports/client/components/PuzzlePage.tsx

```diff
@@ -948,6 +961,75 @@ const PuzzlePageMultiplayerDocument = React.memo((props: PuzzlePageMultiplayerDo
   );
 });
 
+const PuzzleDeletedModal = ({
+  puzzleId, huntId, replacedBy,
+}: { puzzleId: string, huntId: string, replacedBy?: string }) => {
+  const canUpdate = useTracker(() => userMayWritePuzzlesForHunt(Meteor.userId(), huntId), [huntId]);
+
+  const replacementLoading = useSubscribe('mongo.puzzles.allowingDeleted', { _id: replacedBy });
+  const loading = replacementLoading();
+
+  const replacement = useTracker(() => Puzzles.findOne(replacedBy), [replacedBy]);
```

Comment:
This is a `findOne`, but you subscribed with `allowingDeleted` above -- did you want `findOneAllowingDeleted` here?